src/test/groovy/com/stormpath/tck/login/LoginIT.groovy

```diff
@@ -519,8 +528,8 @@ class LoginIT extends AbstractIT {
 
         XmlPath doc = getHtmlDoc(response)
 
-        Node header = HtmlUtils.findTagWithAttribute(doc.getNodeChildren("html.body"), "div", "class", "header")
-        assertThat(getNodeText(header, false), not(isEmptyOrNullString()))
+        Node header = HtmlUtils.findTagWithAttribute(doc.getNodeChildren("html.body"), "span", "id", "status")
+        assertEquals(getNodeText(header, false), "Click HereYour account verification email has been sent! Before you can log into your account, you need to activate your account by clicking the link we sent to your inbox. Didn't get the email?")
```

Comment:
Is this text correct where there's no space between "Click Here" and "Your account"?